msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md

```diff
@@ -23,18 +24,25 @@ Teams AI library is built on top of the Bot Framework SDK and uses its fundament
 > [!NOTE]
 > The adapter class that handles connectivity with the channels is imported from [Bot Framework SDK](/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0#the-bot-adapter&preserve-view=true).
 
+# [JavaScript](#tab/javascript4)
+
 [Sample code reference](https://github.com/microsoft/teams-ai/blob/main/js/samples/04.ai.a.teamsChefBot/src/index.ts#L9)
 
-```typescript
+```javascript
 // Import required bot services.
 // See https://aka.ms/bot-services to learn more about the different parts of a bot.
 import {
     CloudAdapter,
     ConfigurationBotFrameworkAuthentication,
     ConfigurationServiceClientCredentialFactory,
-    MemoryStorage
+    MemoryStorage,
+    TurnContext
 } from 'botbuilder';
 
+// Read botFilePath and botFileSecret from .env file.
+const ENV_FILE = path.join(__dirname, '..', '.env');
+config({ path: ENV_FILE });
+
```

Comment:
Needs to be updated to latest changes

Reply:
Hi @corinagum ,

I've checked the breaking chnages sample for Teams Chef bot and also the main sample, both the places the code is similar to what i've updated. 

- https://github.com/microsoft/teams-ai/blob/breaking-changes-1/js/samples/04.ai.a.teamsChefBot/src/index.ts#L19
- https://github.com/microsoft/teams-ai/blob/main/js/samples/04.ai.a.teamsChefBot/src/index.ts#L19
- https://github.com/microsoft/teams-ai/blob/breaking-changes-2/js/samples/04.ai.a.teamsChefBot/src/index.ts#L19


I may be missing something, can you share the latest changes for this code.

thanks,
Yash